site2/docs/concepts-messaging.md

```diff
@@ -396,6 +397,57 @@ producer = client.create_producer(topic='my-topic', batching_type=pulsar.Batchin
 > * You need to specify a key or orderingKey for messages.
 > * You cannot use cumulative acknowledgment with Key_Shared type.
 
+### Subscription mode
```

Comment:
This heading should be in the same format of `Subscription types`. I suggest we use `Subscription modes`.

Reply:
Subscription modes